### DISCUSSION TITLE
Dropdown panel menu adding hide option for viewers

### DISCUSSION THREAD

**Discussion by KOCIU22** [grafana/grafana#69375] [Dashboards, Panels & Visualizations]
Hello

Can u add option to hide dropdown menu panel for viewers ? I found old topic on grafana forum people try to modify this by yourself  in code i think it should be default dasboard option here is link : https://community.grafana.com/t/disable-view-share-and-more-menu-from-graph/26572?page=2 and below screen

![image](https://github.com/grafana/grafana/assets/78568161/fe757632-f1fb-4834-b916-bb690893844b)

Best Regards Luke

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82635. Feel free to continue the discussion around this there. Thank you!
